ee/codegen/src/__test__/nodes/__snapshots__/inline-prompt-node-retry.test.ts.snap

```diff
@@ -0,0 +1,226 @@
+        )
+    }
+    display_data = NodeDisplayData(
+        position=NodeDisplayPosition(x=0, y=0), width=None, height=None
+    )
+"
+`;
+
+exports[`InlinePromptRetryNode > basic > getNodeFile 1`] = `
+"from vellum import JinjaPromptBlock, PromptParameters
+from vellum.workflows.nodes.core.retry_node.node import RetryNode
+from vellum.workflows.nodes.displayable import InlinePromptNode
+
+from ..inputs import Inputs
+
+
+@RetryNode.wrap(max_attempts=3)
+class PromptNode(InlinePromptNode):
+    ml_model = "gpt-4o-mini"
+    blocks = [
+        JinjaPromptBlock(template="""Summarize what this means {{ INPUT_VARIABLE }}"""),
+    ]
+    prompt_inputs = {
+        "text": Inputs.text,
+    }
+    parameters = PromptParameters(
+        stop=[],
+        temperature=0,
+        max_tokens=1000,
+        top_p=1,
+        top_k=0,
+        frequency_penalty=0,
+        presence_penalty=0,
+        logit_bias={},
+        custom_parameters={},
+    )
+"
+`;
+
+exports[`InlinePromptRetryNode > basic retry adornment and try adornment > getNodeDisplayFile 1`] = `
+"from uuid import UUID
+
+from vellum_ee.workflows.display.nodes import (
+    BaseInlinePromptNodeDisplay,
+    BaseTryNodeDisplay,
+)
+from vellum_ee.workflows.display.nodes.types import (
+    NodeOutputDisplay,
+    PortDisplayOverrides,
+)
+from vellum_ee.workflows.display.vellum import NodeDisplayData, NodeDisplayPosition
+
+from ...nodes.prompt_node import PromptNode
+
+
+@BaseTryNodeDisplay.wrap(error_output_id=UUID("e7a1fbea-f5a7-4b31-a9ff-0d26c3de021f"))
+class PromptNodeDisplay(BaseInlinePromptNodeDisplay[PromptNode]):
+    label = "Prompt Node"
+    node_id = UUID("7e09927b-6d6f-4829-92c9-54e66bdcaf80")
+    output_id = UUID("2d4f1826-de75-499a-8f84-0a690c8136ad")
+    array_output_id = UUID("771c6fba-5b4a-4092-9d52-693242d7b92c")
+    target_handle_id = UUID("3feb7e71-ec63-4d58-82ba-c3df829a2948")
+    prompt_input_ids_by_name = {"text": UUID("7b8af68b-cf60-4fca-9c57-868042b5b616")}
+    node_input_ids_by_name = {"text": UUID("7b8af68b-cf60-4fca-9c57-868042b5b616")}
+    output_display = {
+        PromptNode.Outputs.text: NodeOutputDisplay(
+            id=UUID("2d4f1826-de75-499a-8f84-0a690c8136ad"), name="text"
+        ),
+        PromptNode.Outputs.results: NodeOutputDisplay(
+            id=UUID("771c6fba-5b4a-4092-9d52-693242d7b92c"), name="results"
+        ),
+    }
+    port_displays = {
+        PromptNode.Ports.default: PortDisplayOverrides(
+            id=UUID("dd8397b1-5a41-4fa0-8c24-e5dffee4fb98")
+        )
+    }
+    display_data = NodeDisplayData(
+        position=NodeDisplayPosition(x=0, y=0), width=None, height=None
+    )
+"
+`;
+
+exports[`InlinePromptRetryNode > basic retry adornment and try adornment > getNodeFile 1`] = `
+"from vellum import JinjaPromptBlock, PromptParameters
+from vellum.workflows.nodes.core import TryNode
+from vellum.workflows.nodes.displayable import InlinePromptNode
+
+from ..inputs import Inputs
+
+
+@TryNode.wrap()
```

Comment:
[nit] are we looking to have both here? That's the impression I get from the test label